docs/mllib-linear-methods.md

```diff
@@ -78,6 +78,10 @@ methods `spark.mllib` supports:
   </tbody>
 </table>
 
+A binary label y is denoted as either +1 (positive) or −1 (negative), which is
```

Comment:
Can i keep it only below statement...?

``Note: The negative label is represented by 0 in spark.mllib instead of −1, to be consistent with multiclass labeling.``